test_files/cast_extends/cast_extends.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @fileoverview Reproduces an issue where tsickle would emit a cast for the "extends" claus, and
+ * Closure would report an error due to the extends expression not resolving to a plain identifier.
```

Comment:
How do transformers trigger this?

Reply:
Transformers are much better at retaining comments and parens - previously we'd just silently drop the whole comment sequence that replaced the cast.